apps.yml

```diff
@@ -4030,7 +4030,7 @@ apps:
     authorized_users: []
     client_id: i1qBrMjJEdKlTNEVgGwc9P0xFxI9cuD8
     display: false
```

Comment:
You have to set `display: true` for the tile to be displayed

Reply:
Fixed in ed01bf844006be2bf409692382fea31d774018ad